tests/voq/test_voq_nbr.py

```diff
@@ -684,11 +882,14 @@ def localport_admindown(self, dut, dut_intf):
 
         """
         logger.info("Admin down port %s/%s", dut.hostname, dut_intf)
-        dut.shutdown(dut_intf)
+        if dut.is_multi_asic:
```

Comment:
we can use `asic.shutdown_interface(`). This will handle single and multi asic

Reply:
Fixed in latest.